### PR TITLE
Fixes Part of #2833: Add missing named argument

### PR DESCRIPTION
**Pull request by arjunrkaushik**
## Explanation
Adds named arguments to AppVersionActivityTest

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by arjunrkaushik** [comment]
@rt4914 Could you please help me figure out why the 3 checks are failing...

File: app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AppVersionActivityTest.kt [hunk 1]
```diff
@@ -196,22 +194,22 @@ class AppVersionActivityTest {
...
 
   private fun launchAppVersionActivityIntent(): ActivityScenario<AppVersionActivity> {
     val intent = AppVersionActivity.createAppVersionActivityIntent(
-      ApplicationProvider.getApplicationContext()
+      context = ApplicationProvider.getApplicationContext()
     )
     return ActivityScenario.launch(intent)
   }
 
   private fun launchAdministratorControlsActivityIntent(profileId: Int): Intent {
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AppVersionActivityTest.kt:209]: Rename `profileId ` to `internalProfileId` and also whereever you are using it as named argument.

File: app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AppVersionActivityTest.kt [hunk 2]
```diff
@@ -177,13 +175,13 @@ class AppVersionActivityTest {
   fun testAppVersionActivity_loadFragment_onBackPressed_displaysAdministratorControlsActivity() {
     ActivityScenario.launch<AdministratorControlsActivity>(
       launchAdministratorControlsActivityIntent(
-        0
+        profileId = 0
       )
     ).use {
       testCoroutineDispatchers.runCurrent()
       onView(withId(R.id.administrator_controls_list)).perform(
         scrollToPosition<RecyclerView.ViewHolder>(
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AppVersionActivityTest.kt:183]: This should not be changed. This is the reason why the tests are failing.
`scrollToPosition<RecyclerView.ViewHolder>(3)`

File: app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AppVersionActivityTest.kt [hunk 3]
```diff
@@ -196,22 +194,22 @@ class AppVersionActivityTest {
...
-      Locale.US
+      timestamp = dateTimeTimestamp,
+      locale = Locale.US
     )
   }
 
   private fun launchAppVersionActivityIntent(): ActivityScenario<AppVersionActivity> {
     val intent = AppVersionActivity.createAppVersionActivityIntent(
-      ApplicationProvider.getApplicationContext()
+      context = ApplicationProvider.getApplicationContext()
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AppVersionActivityTest.kt:204]: This is no needed as the value which is `ApplicationProvider.getApplicationContext()` is  self-explanatory.

**Review by rt4914 — CHANGES_REQUESTED** [review]
Thanks. Suggested changes.


Also, in future name your branch something which is more descriptive, it should not contain your name or the issue number.

Suggestion: `named-arguments-app-version-activity-test`

**Comment by arjunrkaushik** [comment]
@rt4914 Thank you so much!!

**Pull request closed without merging**